pixi/src/checks/PageExperienceCheck.js

```diff
@@ -153,7 +155,7 @@ export default class PageExperienceCheck {
                   this.isFastData(fieldData, 'lcp'),
                 ...fieldData,
               }
-            : undefined,
+            : null,
```

Comment:
Any reason for using null here?
I think there are other places where we should change it if we prefer null over undefined.

Reply:
`null` is valid JSON while `undefined` is not, that was my only reasoning. In my opinion this also makes debugging easier: having a value of `null` means that a variable has been evaluated while `undefined` could also mean it has not been evaluated at all.